api/test/pact/pact-tests/wa/searchForCompletableTasks.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { expect } from 'chai';
+import { PactTestSetup } from '../settings/provider.mock';
+import { searchTasks } from "../../pactUtil";
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+const pactSetUp = new PactTestSetup({ provider: 'wa_task_management_api_search_completable', port: 8000 });
+
+
+describe("Task management api, Search for completebale tasks on case event", () => {
+
+    const RESPONSE_BODY = {
+        "tasks": [
+            {
+                "assignee": somethingLike("004b7164-0943-41b5-95fc-39794af4a9fe"),
+                "auto_assigned": somethingLike(true),
+                "case_category": somethingLike("protection"),
+                "case_id": somethingLike("1616666869740714"),
+                "case_name": somethingLike("Bob Smith"),
+                "case_type_id": somethingLike("Asylum"),
+                "created_date": somethingLike("2020-09-05T14:47:01.250542+01:00"),
+                "due_date": somethingLike("2020-09-25T14:47:01.250542+01:00"),
+                "execution_type": somethingLike("Case Management Task"),
+                "id": somethingLike("f782bde3-8d51-11eb-a9a4-06d032acc76d"),
+                "jurisdiction": somethingLike("IA"),
+                "location": somethingLike("765324"),
+                "location_name": somethingLike("Taylor House"),
+                "name": somethingLike("task name"),
+                "region": somethingLike("1"),
+                "security_classification": somethingLike("PUBLIC"),
+                "task_state": somethingLike("assigned"),
+                "task_system": somethingLike("SELF"),
+                "task_title": somethingLike("task name"),
+                "type": somethingLike("wa-task-configuration-api-task"),
+                "warnings": somethingLike(false)
+            }
+        ]
+    }
+
+    const mockRequest = {
+        "case_id": "1616666869740714",
+        "case_jurisdiction": "IA",
+        "case_type": "Asylum",
+        "event_id": "f782bde3-8d51-11eb-a9a4-06d032acc76d"
+    }
+
+    describe("post /task/search-for-completable", () => {
+
+        const jwt = 'some-access-token';
+
+        before(async () => {
+            await pactSetUp.provider.setup()
+            const interaction = {
+                state: "some tasks exists on event for case",
```

Comment:
the provider state for this is "appropriate tasks are returned by search for completable"